src/ui/backend.rs

```diff
@@ -222,9 +222,19 @@ where
         self.terminal.write_styled(&x)
```

Comment:
Can you put a description of this change in the CHANGELOG.md file?

Let's put it as a new feature, under a "Breaking" header. Something in the lines of

> now the option under the cursor is coloured green, and it can be customized via the `selected_option` config.

Reply:
Done in https://github.com/mikaelmello/inquire/pull/88/commits/0078706d8861bbb2f3ff0f2a2c5c91b02e7a9ca0